german/the-black-hack.md

```diff
@@ -101,7 +101,7 @@ The Black Hack verwendet keine Rettungswürfe und lässt die Spieler stattdessen
 
 |              INT               |            WEI            |           CHA           |
 |:------------------------------:|:-------------------------:|:-----------------------:|
-| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Geistige Beeinflussung. |
+| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Bezauberung. |
```

Comment:
Spannend wie unterschiedlich die Wahrnehmung da sein kann :+1: 

Deine Definition deckt sich gut mit meiner spontanen Einschätzung, im speziellen Fall ging es aber auch darum wie das in Verbindung mit der Spezialfähigkeit des Zauberers zu werten ist. Da wurde es dann schwieriger, da das englische Original für mich subjektiv was anderes aussagt.